src/templates/next/components/shared/Notification/Notification.tsx

```diff
@@ -0,0 +1,30 @@
+import { NotificationProps } from "~/common/Notification"
+
+import { BiX } from "react-icons/bi"
+import { useState } from "react"
+import BaseParagraph from "../Paragraph"
+
+const Notification = ({ content: title }: NotificationProps) => {
```

Comment:
nit: I think we don't need to rename the prop here?

Reply:
[88000ab](https://github.com/isomerpages/isomer-components/pull/67/commits/88000ab71881e74a26f77b82c12b38498b6edca1)